Adding Ad Astra mod

Mentalgen opened the pull request:
This adds everything from Ad Astra, including Dimensions. In the future we should probably think about a better way of doing shaders for things like space and no clouds/atmosphere. Right now the Overworld shader causes planets in orbit to be completely black.

SpacEagle17 commented:
You seem to remove some things on block.properties by accident which you didn't mean to remove. Check the diff file again please

Mentalgen commented:
Only issues i see are these lines

hauntedharvest:jack_o_lantern

 redbits:inverted_redstone_torch:lit=true redbits:inverted_redstone_wall_torch:lit=true ad_astra:glowing_desh_pillar

 cinderscapes:scorched_tendrils

create:zinc_ore create:deepslate_zinc_ore \

I need to resolve the conflict anyways so it'll be fixed then.

I removed the partial froglight block ID because i didn't know at the time blocks that light emitting blocks don't need to be on odd id numbers. 

Also removed amalgam ore because it just looks bad in any of the ore ids
